src/java/org/apache/cassandra/cql3/QueryProcessor.java

```diff
@@ -131,6 +129,28 @@ public class QueryProcessor implements QueryHandler
                     DatabaseDescriptor.getPreparedStatementsCacheSizeMiB());
     }
 
+    /**
+     * @return The maximum capacity, in bytes, of the prepared statements cache.  If no capacity is established,
+     * {@link Long#MAX_VALUE} is returned.
+     */
+    public static long getPreparedStatementsCacheMaximumCapacity()
```

Comment:
Trash this. You already have `DatabaseDescriptor.getPreparedStatementsCacheSizeMiB()`

Reply:
removed